sympy/solvers/bivariate.py

```diff
@@ -114,6 +120,45 @@ def _linab(arg, symbol):
     return a, b, x
 
 
+def _power_list(f, *symbols):
+    """
+    Helper function to return powers of variables
+    present in f in list of dictionaries.
```

Comment:
Function documentation is inconsistent with the return types.

Reply:
Yeah _power_list is been issue here. It will be improved.